tensorflow/compiler/tests/random_ops_test.py

```diff
@@ -116,12 +116,22 @@ def testTruncatedNormalIsNotConstant(self):
     def rng(dtype):
       return random_ops.truncated_normal(shape=[2], dtype=dtype)
 
-    self._testRngIsNotConstant(rng, dtypes.float32)
+    # There is no double precision float kernel for truncated normal
+    for dtype in self._random_types() & self.float_types - {np.float64}:
+      self._testRngIsNotConstant(rng, dtype)
 
   def testTruncatedNormalIsInRange(self):
-    count = 10000000
-    # TODO(b/34339814): make this test work with 16 bit float types.
-    for dtype in self._random_types() & {dtypes.float32, dtypes.float64}:
+    count = 1000000
+
+    # There is no double precision float kernel for truncated normal
+    for dtype in self._random_types() & self.float_types - {np.float64}:
+
+      # TODO: CPU backend doesn't produce a valid distribution. Remove this
```

Comment:
The comment says CPU  but the code says both CPU and GPU?

If you're being conservative because you don't have a GPU to run the tests on, can you please add a `TODO(sanjoy)` to enable the test if it passes?

Reply:
sure thing.  you are right about being conservative.  given the comments and code further up the file, it seems likely that it will not work on the GPU.

Reply:
ok - i checked the count thing again and it is ok as it was.  not sure why i thought that numpy wouldn't handle it

Reply:
Can you be more specific about the errors you're seeing on the CPU backend?

 - Do they happen with both half and bfloat16?
 - What asserts fail?